<Exception>
<DateTime>12/14/2015 9:40:24 AM</DateTime>
<SessionID>j2srrxqurdmm4eiclrsitrfe</SessionID>
<Summary>An unhandled exception occured on Monday, December 14, 2015 9:40:24 AM</Summary>
<Info>
<Verb>POST</Verb>
<URL>ustxri00dbamgt/DBAWeb/UpgradeServerForm.aspx</URL>
<HostIP>10.172.202.19</HostIP>
<ReffererURL>ustxri00dbamgt/DBAWeb/UpgradeServerForm.aspx</ReffererURL>
</Info>
<Browser>
<Name>Chrome</Name>
<Version>47.0</Version>
<CookieSupport>True</CookieSupport>
<JSSupport>True</JSSupport>
<Platform>WinNT</Platform>
</Browser>
<User>Guest</User>
<ErrorMessage>System.Net.Mail.SmtpException: Failure sending mail. ---> System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.Mail.SmtpPooledStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   at System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at DBAWebsite.UpgradeServerForm.sendMainTask(Int32 result) in h:\DBAWebsite\UpgradeServerForm.aspx.cs:line 332</ErrorMessage>
<InnerException>System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.Mail.SmtpPooledStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   at System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)</InnerException>

</Exception>

